content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
+      To get started, check out <https://istio.io/latest/docs/setup/getting-started/>
+    {{< /text >}}
 
-{{< text syntax=bash snip_id=istioctl_precheck >}}
-$ istioctl x precheck
-✔ No issues found when checking the cluster. Istio is safe to install or upgrade!
-  To get started, check out <https://istio.io/latest/docs/setup/getting-started/>
-{{< /text >}}
+1. Update the Helm repository:
 
-{{< warning >}}
-[Helm does not upgrade or delete CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) when performing an upgrade. Because of this restriction, an additional step is required when upgrading Istio with Helm.
-{{< /warning >}}
+    {{< text syntax=bash snip_id=update_helm >}}
+    $ helm repo update istio
+    {{< /text >}}
 
-### Manually upgrade the CRDs and Istio base chart
+1. Choose a Revision Name
+    Revisions identify unique instances of the Istio control plane, allowing you to run multiple distinct versions of the control plane simultaneously in a single mesh. It is recommended that revisions stay immutable, that is, that once a control plane is installed with a particular revision name, that installation not be modified, and that the revision name not be reused. Tags, on the other hand, are mutable pointers to revisions. This enables a cluster operator to effect data plane upgrades without the need to adjust any workload labels, simply by moving a tag from one revision to the next. All data planes will connect only to one control plane, specified by the istio.io/rev label (pointing to either a revision or tag), or by the default revision if no istio.io/rev label is present. Upgrading a data plane consists of simply changing the control plane it is pointed to via modifying labels or editing tags.
 
-1. Upgrade the Kubernetes custom resource definitions ({{< gloss >}}CRDs{{</ gloss >}}):
+    Because Revisions are intended to be immutable, we recommend choosing a revision name that corresponds with the version of Istio you are installing, such as `1-22-1`. In addition to choosing a new revision name, you should note your current revision name. You can find this by running
 
-    {{< text syntax=bash snip_id=manual_crd_upgrade >}}
-    $ kubectl apply -f manifests/charts/base/crds
+    {{< text syntax=bash snip_id=list_revisions >}}
+    $ kubectl get mutatingwebhookconfigurations -l 'istio.io/rev,!istio.io/tag' -L istio\.io/rev
+    $ # Store your revision and new revision in variables:
+    $ export REVISION=istio-1-22-1
+    $ export OLD_REVISION=istio-1-21-2
     {{< /text >}}
 
-1. Upgrade the Istio base chart:
+## Upgrade the Kubernetes custom resource definitions ({{< gloss >}}CRDs{{</ gloss >}})
 
-    {{< text syntax=bash snip_id=upgrade_base >}}
-    $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
+    {{< text bash >}}
+    $ kubectl apply -f manifests/charts/base/crds
     {{< /text >}}
```

Comment:
```suggestion
The cluster-wide Custom Resource Definitions (CRDs) must be upgraded prior to the deployment of a new version of the control plane:

{{< text bash >}}
$ kubectl apply -f manifests/charts/base/crds
{{< /text >}}
```

Reply:
Why not just `base` btw?  Do roles ever change?

Reply:
helm skips applying CRDs, so these have to be directly applied.